### ISSUE TITLE
Bug report

### ISSUE THREAD

**Issue by Saloni-Girhepuje** [ethereum/ethereum-org-website#8398]
### Describe the bug

The primary button colour in light mode should be changed from Hex code: 1C1CF5 to Hex code: 1718A5. 
Reasons:
1. Provides better contrast against the background by a contrast ratio of 3 points. - Improved accessibility
2. It matches the colour pallet used in the website illustrations better.- Coherent design system

### To reproduce

1. Go to the Ethereum homepage
2. Access light mode
3. Check the Contrast ratio for the Primary button.

### Expected behavior

The contrast ratio for the **current primary colour** (light mode) Hex code: 1C1CF5 against soft-orange background (from 'A fairer financial system' panel)
<img width="318" alt="image" src="https://user-images.githubusercontent.com/109195419/198684085-3df41f72-3db6-427f-a7cb-5314af3cfe96.png">

The contrast ratio for the **suggested primary colour** (light mode) Hex code: 1718A5 against the same background
<img width="319" alt="image" src="https://user-images.githubusercontent.com/109195419/198684944-9679deba-a4b4-421a-903a-766bfcc71be7.png">

### Screenshots

6.76
The contrast ratio for the **current primary colour** (light mode) Hex code: 1C1CF5 against soft-orange background (from 'A fairer financial system' panel)
<img width="318" alt="image" src="https://user-images.githubusercontent.com/109195419/198684085-3df41f72-3db6-427f-a7cb-5314af3cfe96.png">
- Contrast ratio: 6.76
- WCAG check: AA pass

The contrast ratio for the **suggested primary colour** (light mode) Hex code: 1718A5 against the same background
<img width="319" alt="image" src="https://user-images.githubusercontent.com/109195419/198684944-9679deba-a4b4-421a-903a-766bfcc71be7.png">
- Contrast ratio: 10.02
- WCAG check: AAA pass

**Comment by Mohiiit** [comment]
I would like to work on this.

**Comment by nloureiro** [comment]
@Saloni-Girhepuje I love to see design issues open, thank you for this (next time add a design label to it, to be a bit quicker to circle back to it ;) )

I will close this issue for two reasons:
1) we aim for a AA, AAA is welcome but AA is good enough for our audience.
2) we are in the process of creating a Design system that will be the base for all pages on ethereum.org.


This Design system is not finished yet but we already made the color scheme that will be implemented on Q1.
There will be two different moments regarding the colors:
1) implement an improved color scheme with clear rules on where to use each color, you can check it on figma and I incite you to comment there with your take/suggestions/improvements
Link to figma colors: https://www.figma.com/file/NrNxGjBL0Yl1PrNrOT8G2B/ethereum.org-Design-System?node-id=101%3A538&t=guK4RourloAwzhDQ-1

2) I want to start a bigger discussion on use a blue/orange primary color. we are not here yet but next year I have the goal of change this base colors to something more more appealing and close to the brand (see the glyph on the header)

Any other take,  please let me know what's on your mind bout this.
